README.md

```diff
@@ -226,3 +227,7 @@ _Tutorials for participating in the ICON ecosystem_
 ## Templates
 _Templates for projects or other useful things_
 - [Template repository](https://github.com/icon-community/template-repository) - Template repository for a new project, including README, Issue Templates, PR Template, LICENSE, Release message auto-instructions, CONTRIBUTING, and CODE_OF_CONDUCT files
+
+## Zero-knowledge proofs
```

Comment:
Needs to be a subcategory of the 'projects' section

Reply:
done